### PR TITLE
zh, en: update description of evict leader timeout of TiKV

### PR DESCRIPTION
**Pull request by KanShiori**
### What is changed, added, or deleted? (Required)
update description of evict leader timeout of TiKV
Ref: https://github.com/pingcap/tidb-operator/pull/4075

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/configure-a-tidb-cluster.md [hunk 1]
```diff
@@ -490,9 +490,9 @@ When Kubernetes deletes the TiDB Pod, it also removes the TiDB node from the ser
...
 
-During TiKV upgrade, TiDB Operator evicts all Region leaders from TiKV Pod before restarting TiKV Pod. Only after the eviction is completed (which means the number of Region leaders on TiKV Pod drops to 0) or the eviction exceeds the specified timeout (10 minutes by default), TiKV Pod is restarted. 
+During TiKV upgrade, TiDB Operator evicts all Region leaders from TiKV Pod before restarting TiKV Pod. Only after the eviction is completed (which means the number of Region leaders on TiKV Pod drops to 0) or the eviction exceeds the specified timeout (1500 minutes by default), TiKV Pod is restarted. 
 
-If the eviction of Region leaders exceeds the specified timeout, restarting TiKV Pod causes issues such as failures of some requests or more latency. To avoid the issues, you can configure the timeout `spec.tikv.evictLeaderTimeout` (10 minutes by default) to a larger value. For example:
+If the eviction of Region leaders exceeds the specified timeout, restarting TiKV Pod causes issues such as failures of some requests or more latency. You can configure the timeout by `spec.tikv.evictLeaderTimeout` (1500 minutes by default). For example:
```
Comment (qiancai) [inline en/configure-a-tidb-cluster.md:495]:
```suggestion
If the eviction of Region leaders exceeds the specified timeout, restarting TiKV Pod causes issues such as failures of some requests or more latency. To update the timeout, you can configure `spec.tikv.evictLeaderTimeout` (1500 minutes by default). For example:
```

**Review by qiancai — COMMENTED** [review]
@KanShiori  I made one comment. PTAL.

File: zh/configure-a-tidb-cluster.md [hunk 2]
```diff
@@ -475,9 +475,9 @@ Kubernetes 在删除 TiDB Pod 的同时，也会把该 TiDB 节点从 Service 
 
 ### 配置 TiKV 平滑升级
 
-TiKV 升级过程中，在重启 TiKV Pod 之前，TiDB Operator 会先驱逐 TiKV Pod 上的所有 Region leader。只有当驱逐完成（即 TiKV Pod 上的 Region leader 个数为 0）或者驱逐超时（默认 10 分钟）后，TiKV Pod 才会重启。
+TiKV 升级过程中，在重启 TiKV Pod 之前，TiDB Operator 会先驱逐 TiKV Pod 上的所有 Region leader。只有当驱逐完成（即 TiKV Pod 上的 Region leader 个数为 0）或者驱逐超时（默认 1500 分钟）后，TiKV Pod 才会重启。
```
Comment (csuzhangxc) [inline zh/configure-a-tidb-cluster.md:478]: 我们要不要说一下是从哪个版本开始有这个时间的变更

Comment (KanShiori) [reply zh/configure-a-tidb-cluster.md:478]: 2ae3369

**Pull request closed without merging**